src/Api/Controllers/AppsKeysController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Canvas\Api\Controllers;
+
+use Canvas\Http\Response;
+use Canvas\Models\AppsKeys;
+
+/**
+ * Class CompaniesController.
+ *
+ * @package Canvas\Api\Controllers
+ *
+ * @property Users $userData
+ * @property Request $request
+ */
+class AppsKeysController extends BaseController
+{
+    /*
+     * fields we accept to create
+     *
+     * @var array
+     */
+    protected $createFields = [
+        'client_id',
+        'client_secret_id',
+        'apps_id',
+        'users_id',
+    ];
+
+    /*
+     * fields we accept to create
+     *
+     * @var array
+     */
+    protected $updateFields = [
+        'last_used_date',
+    ];
+
+    /**
+     * set objects.
+     *
+     * @return void
+     */
+    public function onConstruct()
+    {
+        $this->model = new AppsKeys();
+        $this->model->client_id = bin2hex(random_bytes(64));
```

Comment:
@rwhite27 you have this to create on each CRUD?

I would prefer to use processInput for this